docs/loading/Spark-connector-starrocks.md

```diff
@@ -163,15 +165,42 @@ Construct data in memory and load data into the StarRocks table.
     // 2. Write to starrocks with the format "starrocks",
     // and replace the options with your own.
     df.write.format("starrocks")
-        .option("starrocks.fe.http.url", "127.0.0.1:8038")
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/2491f230-daac-409c-928f-0ef2c9927151)
write "what" and your own "what"?